test/models/application_test.rb

```diff
@@ -6,4 +6,39 @@ class ApplicationTest < ActiveSupport::TestCase
 
     assert application.intake?
   end
+
+  test "application intake_completed transitions from intake to follow_up" do
+    application = Application.new
+    application.intake_completed
+
+    assert application.follow_up?
+  end
+
+  test "application follow_up_completed transitions from follow_up to in_progress" do
+    application = Application.new(state: 'follow_up')
+    application.follow_up_completed
+
+    assert application.in_progress?
+  end
+
+  test "application review transitions from in_progress to in_review" do
+    application = Application.new(state: 'in_progress')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.